ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -131,20 +125,20 @@ def increment_step_and_update_last_reward(self):
         """
         Increment the step count of the trainer and Updates the last reward
         """
-        if len(self.stats["Environment/Cumulative Reward"]) > 0:
+        if self.stats["Environment/Cumulative Reward"]:
             mean_reward = np.mean(self.stats["Environment/Cumulative Reward"])
             self.policy.update_reward(mean_reward)
         self.policy.increment_step()
         self.step = self.policy.get_current_step()
 
     def construct_curr_info(self, next_info: BrainInfo) -> BrainInfo:
         """
-        Constructs a BrainInfo which contains the most recent previous experiences for all agents info
+        Constructs a BrainInfo which contains the most recent previous experiences for all agents
         which correspond to the agents in a provided next_info.
         :BrainInfo next_info: A t+1 BrainInfo.
         :return: curr_info: Reconstructed BrainInfo to match agents of next_info.
         """
-        visual_observations = [[]]
+        visual_observations = []
```

Comment:
Is this correct? We do `visual_observations[i].append(...)` below, but never resize this directly. (I think it may have been broken before for >1 visual observations too)

Reply:
This was changed in the last commit (with types).